deploy/generic-k8s/ecommerce-app/frontend.yaml

```diff
@@ -69,10 +71,11 @@ metadata:
   name: frontend
 spec:
   ports:
-  - port: 3000
-    protocol: TCP
-    targetPort: 3000
+    - port: 80
+      protocol: TCP
+      targetPort: 3000
+      name: http
   selector:
     service: frontend
     app: ecommerce
-  type: ClusterIP
+  type: LoadBalancer
```

Comment:
For the generic K8s deployment I wouldn't change this to a LoadBalancer, as if people run this in minikube or something else, this is a bit more cumbersome to get it working. I would stick to ClusterIP or NodePort for the default, and maybe add a comment for LoadBalancer

Reply:
I think I'll work this into the HELM chart so you have the option for either.